pandas/tests/frame/methods/test_drop.py

```diff
@@ -268,6 +269,24 @@ def test_drop_non_empty_list(self, index, drop_labels):
         with pytest.raises(KeyError, match="not found in axis"):
             DataFrame(index=index).drop(drop_labels)
 
+    @pytest.mark.parametrize("empty_listlike", [[], DatetimeIndex([])])
+    def test_drop_empty_listlike_non_unique_datetime_index(self, empty_listlike):
+        # GH#27994
+        data = {
+            "column_a": [5, 10],
+            "column_b": ["one", "two"]
+        }
+        index = [
+            Timestamp("2021-01-01"),
+            Timestamp("2021-01-01")
+        ]
```

Comment:
You could reduce line breaks here

Reply:
Done.  black actually reformats the 'data' dictionary to one line as well now.